spec/support/request_helper.rb

```diff
@@ -0,0 +1,20 @@
+module RequestHelper
```

Comment:
Are these changes just piggybacking on this PR?

Reply:
Yeah, I added a helper to authenticate a user in the specs. The analytics request specs require you be authed.